src/types.rs

```diff
@@ -0,0 +1,60 @@
+use crate::datetime_utils::deserialize_datetime;
+use crate::proof_type::SnarkifyProofType;
+use crate::task_state::SnarkifyTaskState;
+use chrono::{DateTime, Utc};
+use scroll_proving_sdk::prover::proving_service::ProveRequest;
+use scroll_proving_sdk::prover::types::CircuitType;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Debug)]
+pub struct SnarkifyGetVkResponse {
+    /// Base64 encoded verification key.
+    pub vk: String,
+}
```

Comment:
I really don't want to torture you with this again, but it's still unclear what might lie here! I.e. encoding is great (really), but what is the format of the encoded data inside? If it's arbitrary, what does it depend on? etc

Reply:
added the usage of the vk.